pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -757,9 +783,88 @@ func (gce *GCECloud) EnsureLoadBalancer(clusterName string, apiService *api.Serv
 
 	status := &api.LoadBalancerStatus{}
 	status.Ingress = []api.LoadBalancerIngress{{IP: ipAddress}}
+
 	return status, nil
 }
 
+func serviceNeedsHealthCheck(service *api.Service) bool {
```

Comment:
Yes, will move to be alongside the annotations themselves in api/service/annotations.go